BREAKING.md

```diff
@@ -236,11 +236,18 @@ The @fluid-experimental/get-container package was deprecated in version 0.39 and
 -   [Support for passing empty string in `IUrlResolver.getAbsoluteUrl` relativeUrl argument in OdspDriverUrlResolverForShareLink and OdspDriverUrlResolver](#Support-for-passing-empty-string-in-IUrlResolver.getAbsoluteUrl-relativeUrl-argument-in-OdspDriverUrlResolverForShareLink-and-OdspDriverUrlResolver)
 -   [Deprecate `ensureContainerConnected()` in `@fluidframework/test-utils`](#deprecate-ensurecontainerconnected-in-fluidframeworktest-utils)
 -   [Deprecate internal connection details from `IConnectionDetails`](#deprecate-internal-connection-details-from-IConnectionDetails)
+[Deprecated ContainerServices in `@fluidframework/aqueduct`](#deprecated-ContainerServices)
```

Comment:
```suggestion
-   [Deprecated ContainerServices in @fluidframework/aqueduct](#deprecated-ContainerServices-in-@fluidframework/aqueduct)
```
Few nits here:
- Added missing list syntax
- We are trying to avoid using syntax highlighting in the titles
- Fixed the link so it should properly link to the notes section below

Reply:
This is the wrong release - you can add these notes under the already-existing "Aqueduct members deprecated" section above.